doc/index.rst

```diff
@@ -48,17 +48,49 @@ For example, the ``'qiskit.aer'`` device with two wires is called like this:
 Backends
 ~~~~~~~~
 
-Qiskit devices have different **backends**, which define which actual simulator or hardware is used by the
-device. Different simulator backends are optimized for different types of circuits. A backend can be defined as
-follows:
+Qiskit devices have different **backends**, which define which actual simulator or hardware is 
+used by the device. We recommend passing in a backend instance rather than a string with the 
+backend name to the devices. 
+
+Different simulator backends are optimized for different purposes. To change what backend is used, 
+a simulator backend can be defined as follows:
+
+.. code-block:: python
+
+    from qiskit_aer import UnitarySimulator
+
+    dev = qml.device('qiskit.aer', wires=<num_qubits>, backend=UnitarySimulator())
+
+.. note::
+
+    For ``'qiskit.aer'``, PennyLane chooses the ``aer_simulator`` as the default backend if no 
+    backend is specified. For more details on the ``aer_simulator``, including available backend 
+    options, see `Qiskit Aer Simulator documentation <https://qiskit.github.io/qiskit-aer/stubs/qiskit_aer.AerSimulator.html#qiskit_aer.AerSimulator.run>`_.
+
+    Occassionally, you may see others pass in a string as a backend. For example:
+
+    .. code-block:: python
+
+        dev = qml.device('qiskit.aer', wires=<num_qubits>, backend='unitary_simulator')
+
+    At the time of writing, this is still functional. However, this will soon be deprecated and may 
+    not function as intended. To ensure accurate results, we recommend passing in a backend instance.
```

Comment:
This section of the note is specific enough to the `aer` device (and to a UI we are moving away from) that I would suggest moving it to the `aer` device page